src/_guides/language/effective-dart/documentation.md

```diff
@@ -267,11 +267,56 @@ int weekday;
 int get checkedCount => ...
 {% endprettify %}
 
+### PREFER starting boolean variables or properties with "Whether" followed by noun or gerund phrases.
+
+The doc comment should clarify the states this variable represents. This is
+true even for getters which may do calculation or other work. What the caller
+cares about is the *result* of that work, not the work itself.
+
+{:.good}
+<?code-excerpt "docs_good.dart (noun-phrases-for-non-boolean-var-etc)"?>
+{% prettify dart tag=pre+code %}
+/// Whether the modal is currently displayed to the user.
+bool isVisible;
+
+/// Whether the modal should confirm the user's intent on navigation.
+bool get shouldConfirm => ...
+
+/// Whether resizing the current browser window will also resize the modal.
+bool get canResize => ...
+{% endprettify %}
+
+{{site.alert.note}}
+  This guideline intentionally doesn't include using "Whether or not". In many
+  cases, usage of "or not" with "whether" is superfluous and can be omitted,
+  especially when used in this context.
+{{site.alert.end}}
+
+### DON'T write documentation for both the getter and setter of a property.
```

Comment:
I like adding this as a standalone guideline, makes it easier to discover and point to. Thanks :)